src/widgets/modals/LoginModal.js

```diff
@@ -144,9 +145,20 @@ export class LoginModal extends React.Component {
               style={globalStyles.authFormLogo}
               source={require('../../images/logo_large.png')}
             />
-            <View style={globalStyles.horizontalContainer}>
-              <Text style={[globalStyles.authFormTextInputStyle, localStyles.syncSiteName]}>
-                {settings.get(SETTINGS_KEYS.SYNC_SITE_NAME)}
+            <View style={[globalStyles.verticalContainer, { flex: 1 }]}>
```

Comment:
On my emulator I felt there was a bit to much vertical space:

![image](https://user-images.githubusercontent.com/35858975/100167049-063db400-2f23-11eb-8d42-510375546393.png)


So I gave it a spin trying to reduce it :
![image](https://user-images.githubusercontent.com/35858975/100167007-e4dcc800-2f22-11eb-81b5-1c65e8ca24e1.png)

I dunno.


Also removed the `store`, `site` because I thought they were a bit ugly, but I think that could be confusing so I am not sure if thats a good idea.



I just did it like this:

```
 <Image
   resizeMode="contain"
   style={globalStyles.authFormLogo}
   source={require('../../images/logo_large.png')}
 />
 <Text style={[globalStyles.authFormTextInputStyle, { flex: 0 }]}>
  {settings.get(SETTINGS_KEYS.SYNC_SITE_NAME)}
 </Text>
 <Text style={[globalStyles.authFormTextInputStyle, { flex: 0, marginVertical: 10 }]}>
 {
  UIDatabase.objects('Name').filtered(
  'id == $0',
  settings.get(SETTINGS_KEYS.THIS_STORE_NAME_ID)
  )[0].name
 }
</Text>
```

Reply:
Hmm yeah, the vertical space looks much worse on your emulator than it did on mine! I'll have to set up another device for testing purposes I think. I dunno about the text, are real site names quite obviously distinct to users from store names? If so then yeah it's probably redundant and I'll just steal your implementation 🤭

Reply:
I agree with @joshxg on the spacing. I haven't flexbox'd for literally years, but there should be a props on the container and items that determine how they space (e.g. spread out evenly in the container, smallest possible from the top, stretch to fit w/ even spacing...).

When ever I'm confused I end up playing with this website https://cvan.io/flexboxin5/. Is web flexbox but it's close enough to figure out what you want.

Reply:
@joshxg Regarding the label on the values I agree that it isn't the prettiest but also think it's ambiguous/random seeming without, so would keep. Dunno if it'd improve much but could use icons maybe? Unfortunately there isn't a pattern I know that I can lean into here haha.